salt/modules/zpool.py

```diff
@@ -537,6 +537,13 @@ def scrub(zpool, stop=False):
         name of storage pool
     stop : boolean
         if true, cancel ongoing scrub
+    pause : boolean
```

Comment:
Hrm. Is it better to have this as a kwarg or as an entirely new function? I think I could go either way but it's worth discussing, I think.

Reply:
scrub already had a kwarg for stop, so adding one for pause seems logical.

This also fits with the underlying command

```
# start the scrub
zfs scrub mypool
# pause the scrub
zfs scrub -p mypool
# let stop the scrub
zfs scrub -s mypool
```